web/client/actions/users.js

```diff
@@ -0,0 +1,250 @@
+/**
+ * Copyright 2015, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const USERMANAGER_GETUSERS = 'USERMANAGER_GETUSERS';
+const USERMANAGER_EDIT_USER = 'USERMANAGER_EDIT_USER';
+const USERMANAGER_EDIT_USER_DATA = 'USERMANAGER_EDIT_USER_DATA';
+const USERMANAGER_UPDATE_USER = 'USERMANAGER_UPDATE_USER';
+const USERMANAGER_DELETE_USER = 'USERMANAGER_DELETE_USER';
+const USERMANAGER_GETGROUPS = 'USERMANAGER_GETGROUPS';
+const API = require('../api/GeoStoreDAO');
+const {get, assign} = require('lodash');
+
+function getUsers(searchText, options) {
+    let params = options && options.params;
+    let start;
+    let limit;
+    if (params) {
+        start = params.start;
+        limit = params.limit;
+    }
+    return (dispatch, getState) => {
+        let text = searchText;
+        let state = getState && getState();
+        if (state) {
```

Comment:
Yes I know that search text and current page should come from the action, but this way the component are not indipendent.  Where you should place them